### PR TITLE
adding wrbsc

### PR DESCRIPTION
**Pull request by kldarek**

### CODE REVIEW

File: datasets/wrbsc/wrbsc.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+        # TODO: This method specifies the datasets.DatasetInfo object which contains informations and typings for the dataset
+        features = datasets.Features(
+            {
+                "sentence1": datasets.Value("string"),
+                "sentence2": datasets.Value("string"),
+                "relationship": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/wrbsc/wrbsc.py:69]: Can you use the ClassLabel feature type here ?
For example
```
"relationship": datasets.ClassLabel(names=["Krzyżowanie_się", "Tło_historyczne", "Źródło", "Dalsze_informacje", "Opis", "Uszczegółowienie", "Parafraza", "Spełnienie", "Zmiana_poglądu", "Tożsamość", "Zawieranie"])
```

You'll need to regenerate the dataset_infos.json file after this change

File: datasets/wrbsc/README.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+### Data Instances
+
+[Needs More Information]
+
+### Data Fields
+
+- `sentence1`: a `string` feature.
+- `sentence2`: a `string` feature.
+- `relationship`: a `string` feature.
```
Comment (lhoestq) [inline datasets/wrbsc/README.md:77]: can you list the values this field can take and describe their meanings please ?

**Review by lhoestq — COMMENTED** [review]
Awesome thank you !

I left a few comments

**Comment by kldarek** [comment]
@lhoestq thanks for the comments! Should be fixed in the latest commit, I assume the CI errors are unrelated.

**Review by lhoestq — APPROVED** [review]
Thank you !
Looks all good now :)

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**